goal_src/goal-lib.gc

```diff
@@ -670,6 +670,11 @@
       )
   )
 
+(defmacro float->int (a)
+  "forcefully casts something as a float to int. be careful."
+  `(the int (the float ,a))
```

Comment:
maybe should be `(the int (the-as float ,a))` in case you start with a `uint32` or something like that?

Reply:
actually, I think I still want this to work properly if you put in an int.